exporters/variables-scss/src/content/generators/cssGenerator.ts

```diff
@@ -0,0 +1,93 @@
+import { DimensionToken, StringToken, Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { CSSHelper } from '@supernovaio/export-helpers';
+import { toPlural } from '../helpers/stringHelper';
+import { tokenVariableName } from '../helpers/tokenHelper';
+
+export const tokenToCSSByType = (
+  token: Token,
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  withParent: boolean,
+): string | null => {
+  if (token.tokenType === TokenType.dimension) {
+    const dimensionToken = token as DimensionToken;
+    const name = tokenVariableName(dimensionToken, tokenGroups, withParent);
+    const value = dimensionToken.value?.measure;
+    const unit = CSSHelper.unitToCSS(dimensionToken.value?.unit);
+
+    return `$${name}: ${value}${unit} !default;`;
+  }
+
+  if (token.tokenType === TokenType.string) {
+    const stringToken = token as StringToken;
+    const name = tokenVariableName(stringToken, tokenGroups, withParent);
+    const value = stringToken.value.text;
+
+    return `$${name}: ${value} !default;`;
+  }
+
+  return null;
+};
+
+export const generateCssFromTokens = (
+  tokens: Token[],
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  hasParentPrefix: boolean,
+): string => {
+  return tokens
+    .map((token) => tokenToCSSByType(token, mappedTokens, tokenGroups, hasParentPrefix))
+    .filter(Boolean)
+    .join('\n');
+};
+
+const generateObjectContent = (tokens: Array<Token>, tokenGroups: Array<TokenGroup>, withParent: boolean): string => {
+  let result = '';
```

Comment:
When you have an accumulator where you store some values during the interaction, it is a sign of a `reduce` function.